baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/UnnecessaryParenthesesTest.java

```diff
@@ -29,7 +29,7 @@
     public void testSwitchExpression() {
         CompilationTestHelper compilationHelper = CompilationTestHelper.newInstance(
                         UnnecessaryParentheses.class, getClass())
-                .setArgs(ImmutableList.of("--enable-preview", "--release", "14"));
+                .setArgs(ImmutableList.of("--enable-preview", "--release", "15"));
```

Comment:
This is to fix this error:
```
invalid source release 14 with --enable-preview
    (preview language features are only supported for release 15)
```

I think that's a restriction by the java-compiler that enable-preview has to be used with the latest release but could not find good documentation on this.

Reply:
Yep I stumbled across this restriction too when writing https://github.com/palantir/gradle-baseline/pull/1549

> javac in JDK $N accepts the --enable-preview flag only in combination with --release $N or -source $N. (For brevity, only --release is mentioned in the rest of this JEP.)

https://openjdk.java.net/jeps/12